parquet-pig/src/main/java/org/apache/parquet/pig/TupleWriteSupport.java

```diff
@@ -174,6 +180,8 @@ private void writeValue(Type type, FieldSchema pigType, Tuple t, int i) {
             bytes = ((DataByteArray)t.get(i)).get();
           } else if (pigType.type == DataType.CHARARRAY) {
             bytes = ((String)t.get(i)).getBytes("UTF-8");
+          } else if (pigType.type == DataType.DATETIME) {
+            bytes = convertDateTimeToInt96((DateTime) t.get(i));
```

Comment:
We want to deprecate int96 in favor of other timestamp types here:
https://github.com/apache/parquet-format/blob/master/LogicalTypes.md#datetime-types
Millis would work right?